basic/src/test/java/org/jboss/hal/testsuite/test/configuration/general/WebAccessTestCase.java

```diff
@@ -26,15 +23,18 @@
  * @author mkrajcov <mkrajcov@redhat.com>
  */
 @RunWith(Arquillian.class)
-@Category(Shared.class)
+@RunAsClient
```

Comment:
Arquillian by default is running in container. If the graphene extension isn't changing it somehow, I believe we should have it for all classes defined as `@RunAsClient`. => it is most likely correct change from my PoV.

Reply:
As I understand testing with Arquillian, you should use @RunAsClient when you want to specify, that the test is not deploying any archive to the container. (https://docs.jboss.org/arquillian/spi/1.0.3.Final/org/jboss/arquillian/container/test/api/RunAsClient.html). Speaking of functionality it does not change a thing

Reply:
As long as we don't use `@Deployment` it's not important IMO. From my POV it's just one more line of code which is not necessary. We should either add the annotation to every test or avoid using it to prevent confusion.

Reply:
I'm just complaining about adding the annotation as part of the commit. By doing it this way you could make an impression that it is somehow needed for the commit logic.